keyboards/b_sides/rev41lp/rules.mk

```
@@ -0,0 +1,18 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = qmk-dfu
```

Are you sure about this? I don't see where it is configured.

---

I'm not, this comes from the import of original code so I didn't touch it. I've only built firmware with catalina bootloader for pro micro.

---

If it's Pro Micro compatible, then it should probably be:
```suggestion
BOOTLOADER = caterina
```

---

It is, I have two built with pro micro!